orbbec_camera/component_descriptions/orbbec_camera_ob_camera_node.json

```
@@ -2,30 +2,60 @@
     "$schema": "https://docs.aica.tech/schemas/1-1-1/component.schema.json",
     "name": "Orbbec Camera",
     "description": {
-        "brief": "Stream color and depth images from an Orbbec camera"
+        "brief": "Stream color and depth images from an Orbbec camera."
     },
     "inherits": "rclcpp::Node",
     "registration": "orbbec_camera::OBCameraNodeDriver",
     "parameters": [
+      {
+        "display_name": "Camera Name",
+        "description": "Name of the camera, used to set the frame names",
+        "parameter_name": "camera_name",
+        "parameter_type": "string",
+        "default_value": "camera"
+      },
+      {
+        "display_name": "Serial Number",
+        "description": "Serial number of the camera, required when using multiple cameras",
+        "parameter_name": "serial_number",
+        "parameter_type": "string",
+        "default_value": null,
+        "optional": true
+      },
+      {
+        "display_name": "USB Port",
+        "description": "USB port of the camera, required when using multiple cameras",
+        "parameter_name": "usb_port",
+        "parameter_type": "string",
+        "default_value": null,
+        "optional": true
+      },
+      {
+        "display_name": "Number of Devices",
+        "description": "Number of devices, required when using multiple cameras",
+        "parameter_name": "device_num",
+        "parameter_type": "int",
+        "default_value": "1"
+      },
       {
         "display_name": "Color Width",
-        "description": "Width of the color image",
+        "description": "Width of the color image. Valid resolution should be one of (1280x800, 1280x720, 640x480, 640x400, 424x240).",
         "parameter_name": "color_width",
         "parameter_type": "int",
         "default_value": null,
         "optional": true
       },
       {
         "display_name": "Color Height",
-        "description": "Height of the color image",
+        "description": "Height of the color image. Check the color width for the available resolution pair values.",
```

```suggestion
        "description": "Height of the color image. Valid resolutions are 1280x800, 1280x720, 640x480, 640x400, 424x240.",
```
I would just copy this here, same for the depth